### ISSUE TITLE
v1.2.5: Compilation broken due to error in STM32AsyncTCP dependency

### ISSUE THREAD

**Issue by chris007de** [khoih-prog/AsyncWebServer_STM32#4]
# Description

As already described in #2 by @jcw, there is an issue with the external dependency of the [STM32AsyncTCP Library](https://github.com/philbowles/STM32AsyncTCP).
In the current version (1.2.5) of AsyncWebServer_STM32, compilation fails with just the same error as described in #2, as the root of the cause is still not fixed upstream, the library.json of the library still states the wrong platform (`  "platforms": "espressif8266"`, https://github.com/philbowles/STM32AsyncTCP/blob/master/library.json#L18).

# Workaround

To get my setup working, I just refer to your fork of the library in my `platform.ini`, as your fork already contains the relevant fix:

```
lib_deps =
    STM32AsyncTCP=https://github.com/khoih-prog/STM32AsyncTCP
    khoih.prog/AsyncWebServer_STM32@^1.2.5
```

# Short-Term Solution

I'd propose that you alter your `library.json` to point to your fork of the lib (https://github.com/khoih-prog/AsyncWebServer_STM32/blob/master/library.json#L42) until it is fixed upstream.

Please let me know if I shall supply this as a PR, I am happy to do so.

# Long-Term solution

PR the relevant fixes in the upstream repo of philbowles.

**Comment by khoih-prog** [comment]
Hi @chris007de 

Thanks for your contribution. As the issue is spanning across several libraries, I have to take over to modify and release the new version.

Your and  [Jean-Claude](https://github.com/jcw) contributions have been noted in [Contributions and Thanks](https://github.com/khoih-prog/AsyncWebServer_STM32#contributions-and-thanks)

Please check to see if there still any other issue.

---

### Releases v1.2.6

1. Fix dependency on unpublished [**STM32AsyncTCP Library**](https://github.com/philbowles/STM32AsyncTCP). Check [Compilation broken due to error in STM32AsyncTCP dependency](https://github.com/khoih-prog/AsyncWebServer_STM32/issues/4) and [how to run one of the examples?](https://github.com/khoih-prog/AsyncWebServer_STM32/issues/2).

**Comment by chris007de** [comment]
Thanks a lot, it's fixed with v1.2.6.
